src/Components/Task.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC, useState } from 'react'
+import {
+    Box,
+    Checkbox,
+    ListItem,
+    Typography
+} from '@material-ui/core'
+
+import DeleteIcon from '@material-ui/icons/Delete';
+
+interface Props {
+    task: {
+        title: string,
+        taskId: number
+    },
+    onRemove: Function
+}
+
+const Task: FC<Props> = ({ task, onRemove }: Props) => {
+
+    const [taskChecked, setTaskChecked] = useState(false)
+
+    function toggleTaskChecked() {
+        setTaskChecked((prevState) => !prevState)
+    }
+
+    return (
+        <ListItem sx={{ mb: '.5rem' }} >
+            <Checkbox
+                inputProps={{ 'aria-label': 'primary checkbox' }}
+                onChange={toggleTaskChecked}
+                data-testid="btnChecked"
+            />
```

Comment:
I suggest reorganizing props:
```suggestion
            <Checkbox
                data-testid="btnChecked"
                inputProps={{ 'aria-label': 'primary checkbox' }}
                onChange={toggleTaskChecked}
            />
```

Reply:
Resolved on 308e3065ba9e4eb3f07ec2b9d51dd3bc08e94738